src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -99,6 +100,10 @@ export const DepartureDetailsScreenComponent = ({
 
   const realtimeText = useRealtimeText(estimatedCallsWithMetadata);
 
+  const isJourneyFinished =
+    vehiclePosition?.vehicleStatus === VehicleStatusEnumeration.Completed ||
+    estimatedCallsWithMetadata.every((e) => e.actualArrivalTime);
```

Comment:
Do we need to keep the check on actualArrivalTime? I would hope vehicle status is reliable enough. Also it would help a bit on performance 🏃💨

Reply:
> Do we need to keep the check on actualArrivalTime? I would hope vehicle status is reliable enough. Also it would help a bit on performance 🏃💨

According to discussions with Entur, this `actualArrivalTime` is a fallback for the status, they are still working on an issue where the last quay is not updated with the final location.

Reply:
Aha, then I understand 👍